### ISSUE TITLE
faster convertToRaw and convertFromRaw? or `getHtmlText()` instead of ?

### ISSUE THREAD

**Issue by terencechow** [facebook/draft-js#1397]
**Do you want to request a *feature* or report a *bug*?**
feature?

**What is the current behavior?**
`convertToRaw` takes about 30ms when text gets to ~3k words. I want to something a la google docs but that means I need websockets. 

Hence I need to `convertToRaw` in an `onChange` handler that then sends the data to the websockets server. The problem is that `convertToRaw` takes 35ms+ when at 3k words and it degrades fast as the words increase. 35ms+ is equivalent to having 20-30 fps in an onchange handler which is not good enough...(I can certainly debounce this, but the more I debounce, the less realtime it becomes...)

In addition to that, `convertFromRaw` takes significantly longer than `convertToRaw` but that's a problem for a later time...

Basically, if `contentState` was already serializable, I wouldn't have any of these issues. A `getHtmlText()` like `getPlainText()` would be a really nice feature. How hard would that be to implement? What causes `convertToRaw` or `convertFromRaw` to take so much time? My 3000 word example didn't have any entities and had only 2 blocks and that still took 35ms.

**What is the expected behavior?**
contentState can be serialized quickly.

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
0.10.1 using chrome on Mac el capitan

**Comment by colinjeanne** [comment]
Rather than send the entire content, can you detect the specific change that occurred and send that?

**Comment by terencechow** [comment]
How could I do that?
